### PR TITLE
Implement rules for CIS OCP Section 3

### PR DESCRIPTION
**Pull request by rhmdnd**
Now that we have a profile and control files for CIS 1.4.0, we can start
wiring up the existing rules.

This commit ports all the existing rules we were using for the CIS
OpenShift profile into the CIS 1.4.0 version.

### CODE REVIEW

File: controls/cis_ocp_1_4_0/section-3.yml
Comment (jhrozek) [inline controls/cis_ocp_1_4_0/section-3.yml:33]: Is there a reason not to use this rule for 3.1.1?

Comment (jhrozek) [reply controls/cis_ocp_1_4_0/section-3.yml:33]: If there is, please remove the 3.1.1 reference from audit_profile_set

**Review by jhrozek — CHANGES_REQUESTED** [review]
either fix the reference in audit_profile_set or use that rule in 3.1.1

File: controls/cis_ocp_1_4_0/section-3.yml
Comment (rhmdnd) [reply controls/cis_ocp_1_4_0/section-3.yml:33]: Updated to include the rule for control 3.2.1 and keeping the references. I misread the comments in the original profile. Good catch.

File: controls/cis_ocp_1_4_0/section-3.yml
Comment (jhrozek) [reply controls/cis_ocp_1_4_0/section-3.yml:26]: automated seems fine to me. I'll ack the patch, just won't merge it just yet, I'll let you decide if this should be merged already.

**Comment by rhmdnd** [comment]
Rebasing to pick up the change to fix some of the build jobs.

**Comment by rhmdnd** [comment]
Rebased and resolved merge conflicts.

File: controls/cis_ocp_1_4_0/section-3.yml
Comment (yuumasato) [reply controls/cis_ocp_1_4_0/section-3.yml:34]: For 3.2.2, shouldn't we also set `var_openshift_audit_profile` to be `WriteRequestBodies`?

Comment (rhmdnd) [reply controls/cis_ocp_1_4_0/section-3.yml:34]: Could we set the `var_openshift_audit_profile` to `WriteRequestBodies` in the CIS profile?

Comment (rhmdnd) [reply controls/cis_ocp_1_4_0/section-3.yml:34]: Good catch, setting that as a profile-level variable in https://github.com/ComplianceAsCode/content/pull/10833

**Pull request merged**
